CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add events `meetingReconnected`, `signalingDropped` and `receivingAudioDropped` to `eventDidReceive` by publishing them as stand alone events. Currently, these events were only included in the meeting history attribute when a meeting event is published. 
 - Add link to SIP Media Application examples in README.
+- Add integration test for MediaCapture features.
```

Comment:
nit: Add a space here after the log. Also, we do not enforce changelog for the integration tests or demo changes now.